content/docs/user-guide/visualization/environments/terrain/terrain-developer-guide/system-usage.md

```diff
@@ -0,0 +1,85 @@
+
+However, the terrain system itself can be queried at any position, not just on the conceptual grids. The `Sampler` type controls exactly _how_ to query the underlying data in relation to these grids.
+
+The `Default` / `Bilinear` Sampler setting uses the query resolution to sample the four grid corners that surround the requested position, and then it performs bilinear filtering to create the resulting output value. For example, in the illustration below, the underlying height data has different values at every 0.25 meter interval. The chart below it shows what happens when querying this data at position 0.75 with different Height Query Resolution settings.
+
+```goat
+40  o                           o
+30  |                           |
+20  |      o      o             |
+10  |      |      |      o      |
+0   +------+------+------+------+
+   0.00   0.25   0.50   0.75   1.00
+```
+
+| Height Query Resolution | Result | Description |
+| - | - | - |
+| 1.0 meters | 40 | Querying at 0.75 will fetch the underlying data at 0.00 (40) and 1.00 (40) and interpolate at 75%. |
+| 0.5 meters | 30 | Querying at 0.75 will fetch the underlying data at 0.50 (20) and 1.00 (40) and interpolate at 50%. |
+| 0.25 meters | 10 | Querying at 0.75 will fetch the underlying data at 0.75 (10) and 1.00 (40) and interpolate at 0%. |
+
+The `Default` Sampler setting is the one that should be used by most systems when querying. However, there are two other sampling methods that are available for specialized use.
+
+The `Clamp` Sampler setting will take the requested query position and 'clamp' it down to the terrain grid before requesting the value. Where the `Bilinear` sampler produces an infinite range of interpolated values for all the positions between grid points, the `Clamp` sampler will always produce the same value for every position between grid points. Because this only fetches one data value, not four, it can run up to 4x faster than the `Default` sampler, but the values returned will only match the rendering and physics representations when queried directly on the grid points. For that reason, this setting should only be used when querying directly on terrain grid points.
+
+The `Exact` Sampler setting will take the requested query position and pass it directly down to the underlying data source, ignoring the terrain grids entirely. Similar to `Clamp`, this is 4x faster than `Default`, but will only produce matching values when queried directly on the grid points. This setting should also only be used when querying directly on terrain grid points, but it may serve some alternate purpose if there's ever a reason to query the underlying terrain data directly without regards to the terrain grid.
```

Comment:
My suggestion reworks your last sentence -- I wanted to remove "but" because I think this is an "and" rather than a "but" situation. I also attempt to streamline the last part, such as removing unnecessary words. 
```suggestion
The `Exact` Sampler setting takes the requested query position and passes it directly down to the underlying data source, ignoring the terrain grids entirely. Similar to `Clamp`, this is 4x faster than `Default`, but only produces matching values when it's queried directly on the grid points. You should only use this setting when you are querying directly on the terrain grid points. Alternatively, `Exact` may be useful if you want to directly query the underlying terrain data, without regards to the terrain grid.
```

Reply:
Hmm, how about this instead?

```suggestion
The `Exact` Sampler setting takes the requested query position and passes it directly down to the underlying data source, ignoring the terrain grids entirely. Similar to `Clamp`, this is 4x faster than `Default`, but only produces matching values when it's queried directly on the grid points. You should not use it to query positions outside of the grid points, since the data may not align with the rendering or physics representations. However, the feature is available for those who want it.
```

The basic points are that the data from `Exact` is useful when queried directly on the grid points, _probably_ isn't useful anywhere else, but _maybe_ someone will have a neat way to use it and we don't want to prevent them from doing it.

Reply:
Yes, sounds good!